Sources/NIO/ChannelPipeline.swift

```diff
@@ -440,29 +477,41 @@ public final class ChannelPipeline: ChannelInvoker {
     ///     - handlerType: The type of the handler to search for.
     /// - returns: the `EventLoopFuture` which will be notified once the the operation completes.
     public func context<Handler: ChannelHandler>(handlerType: Handler.Type) -> EventLoopFuture<ChannelHandlerContext> {
```

Comment:
This function should be `@inlinable`, or the other inlinability annotation won't help us at all.

Reply:
Oof, good catch.